src/cpuid/src/template/mod.rs

```diff
@@ -6,19 +6,125 @@ pub mod intel;
 
 use std::collections::HashSet;
 
+use arch_gen::x86::msr_index::*;
 use kvm_bindings::CpuId;
 
-use crate::common::{get_vendor_id_from_cpuid, VENDOR_ID_INTEL};
+use crate::common::{get_vendor_id_from_cpuid, VENDOR_ID_AMD, VENDOR_ID_INTEL};
```

Comment:
nit: `VENDOR_ID_AMD` is not still used in this commit c9b195eaaf34d21320d7568d5a3d97da7ac8fab3.

Reply:
done